plugin-gradle/src/main/java/com/diffplug/gradle/spotless/JavaExtension.java

```diff
@@ -74,6 +77,9 @@ public class ImportOrderConfig {
 		final File importOrderFile;
 
 		boolean wildcardsLast = false;
+		boolean semanticSort = true;
```

Comment:
We should not change the default behavior for existing users of the `importOrder` step.